### ISSUE TITLE
Command navigation in a11y button should go to the partial command on last ctrl+down

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#188926]
Repro:

1. Enable a11y mode
2. Run some commands
3. Shift+tab
4. ctrl+up a few times
5. ctrl+down a few times, 🐛 the last command navigated to is the last executed command, it should be the partial command being written
